package.json

```diff
@@ -19,18 +19,18 @@
   "author": "",
   "license": "MIT",
   "dependencies": {
-    "@types/bitcore-lib": "^0.15.0",
-    "babel-cli": "^6.26.0",
-    "babel-plugin-module-resolver": "^3.0.0",
-    "bitcore-lib": "^0.15.0",
-    "node-fetch": "^1.7.3",
-    "protobufjs": "^6.8.0"
+    "babel-cli": "6.26.0",
+    "babel-plugin-module-resolver": "3.0.0",
+    "bitcore-lib": "0.15.0",
+    "node-fetch": "2.1.2",
```

Comment:
There's a separate PR by CodeTree for upgrading node-fetch to the latest version. I usually rather make more atomic changes in case something goes wrong, so it's easier to spot what happened.

Also: we should probably update the @types/node-fetch, if we update node-fetch.

Reply:
good catch. I was planning on deleting the greenkeeper PR since it would revert back to an unpinned version. What do you think?

Reply:
Yep, probably easier to just delete it and make a new one.